conf/cmi/field.field.node.page.field_content.yml

```diff
@@ -58,11 +56,8 @@ settings:
       image:
         enabled: true
         weight: 2
-      link:
-        enabled: true
-        weight: 6
       list_of_links:
-        enabled: false
+        enabled: true
```

Comment:
Or this?

Reply:
These must've come from some old db settings. I don't remember changing them..